src/Tools/dotnet-monitor/LoggingExtensions.cs

```diff
@@ -144,6 +144,12 @@ internal static class LoggingExtensions
                 logLevel: LogLevel.Information,
                 formatString: nameof(ConfigurationKeys.ApiAuthentication) + " settings have changed.");
 
+        private static readonly Action<ILogger, string, string, Exception> _logTempKey =
+            LoggerMessage.Define<string, string>(
+                eventId: new EventId(23, "LogTempApiKey"),
+                logLevel: LogLevel.Information,
+                formatString: "Generated one-time-use ApiKey for dotnet-monitor, use the following header for authorization:{0}Authorization: MonitorApiKey {MonitorApiKey}");
```

Comment:
Unnecessary {0}?

Reply:
It's used to inject an Environment.NewLine into the string. See `LoggingExtensions.LogTempKey`.